internal/venice-common/src/main/java/com/linkedin/venice/serialization/avro/AvroProtocolDefinition.java

```diff
@@ -154,9 +154,9 @@ public enum AvroProtocolDefinition {
   PARTICIPANT_MESSAGE_SYSTEM_STORE_VALUE(1, ParticipantMessageValue.class),
 
   /**
-   * Response record for admin request v1.
+   * Response record for admin request.
    */
-  SERVER_ADMIN_RESPONSE_V1(1, AdminResponseRecord.class),
+  SERVER_ADMIN_RESPONSE(2, AdminResponseRecord.class),
```

Comment:
I don't get this; where is version 2?

Reply:
https://github.com/linkedin/venice/blob/main/internal/venice-common/src/main/resources/avro/AdminResponseRecord/v2/AdminResponseRecord.avsc